src/main/java/org/ossgang/commons/observable/mappers/buffer/ConcurrentCircularBuffer.java

```diff
@@ -0,0 +1,78 @@
+package org.ossgang.commons.observable.mappers.buffer;
```

Comment:
I think this is potentially a very useful class in multiple use cases, I would not hide it deep in the hierarchy of observable :-)
What about creating another top-level package like "collections"?

Reply:
Fine for me

Reply:
Indeed, I thought the same ... I moved it.

Reply:
Thanks!